README.md

```diff
@@ -1 +1,72 @@
 # babel-plugin-transform-react-pug
+
+The official means by which you can use pug in your react components, replaces the use of react-jade when moving to pug.
+
+This plugin transforms the pug inside of your react components.
+
+## Installation
+```
+npm install babel-plugin-transform-react-pug
```

Comment:
Also `babel-plugin-transform-react-jsx` and add the `--save-dev` option to the end of this.